CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## 1.3.1 - 2020-05-XX
+### Added
+- Url API now can retrieve response header value attached on each generated sample included a response error.
```

Comment:
I would rephrase as :
Url API can now include all the response headers in the resulting sample. Set `return_headers` to true to enable.

Reply:
👍